### ISSUE TITLE
Add check for WP_UNINSTALL_PLUGIN

### ISSUE THREAD

**Issue by froger-me** [WordPress-Coding-Standards/WordPress-Coding-Standards#1338]
Hello,

I'd like to suggest the following:

How about adding a notice if ` ! defined( 'WP_UNINSTALL_PLUGIN' )` is not present in `uninstall.php` files?

**Comment by GaryJones** [comment]
I'd be fine with it in general, but it may be that `uninstall.php` actually just requires some other files, and it's those files which have the definition check in it (though that would be somewhat daft).

If there's no `uninstall.php`, then the sniff just continues.

**Comment by froger-me** [comment]
@GaryJones agreed this could happen, agreed it would be somehow unorthodox. Maybe a warning would suffice instead of a downright error?

And yes, @dingo-d, this check would happen only in uninstall.php at the root of the plugin folder (as per the Plugin Development Guidelines it's how it should be done when uninstalling with included file(s), if I'm not mistaken).
